servers/rendering/renderer_rd/environment/gi.h

```diff
@@ -755,7 +755,7 @@ class GI : public RendererGI {
 		SHADER_SPECIALIZATION_HALF_RES = 1 << 0,
 		SHADER_SPECIALIZATION_USE_FULL_PROJECTION_MATRIX = 1 << 1,
 		SHADER_SPECIALIZATION_USE_VRS = 1 << 2,
-		SHADER_SPECIALIZATION_VARIATIONS = 0x07,
+		SHADER_SPECIALIZATION_VARIATIONS = 8,
```

Comment:
Does this change go with the other commented out VRS changes?

Reply:
No, that was just a mistake of mine on the VRS PR. I really should open a separate PR for this but seeing I'm expecting this to be merged soonish.

Basically without this fix the shader specialisation with half res, multiview and vrs all on wouldn't work. An unlikely combination :)